sdk/storage/azure_storage_blob/src/generated/clients/blob_client.rs

```diff
@@ -40,11 +40,10 @@ use azure_core::{
 };
 use std::sync::Arc;
 
-#[tracing::client]
 pub struct BlobClient {
     pub(crate) blob_name: String,
     pub(crate) container_name: String,
-    pub(crate) endpoint: Url,
+    pub(crate) blob_url: Url,
```

Comment:
Generated code change #1: `endpoint` -> `blob_url`. We can maintain all other fields because we return these struct fields directly from our convenience layer for the getters